docs/requirements.txt

```diff
@@ -73,6 +73,10 @@ mdurl==0.1.2
     # via markdown-it-py
 myst-parser==4.0.0
     # via ops (pyproject.toml)
+ops==2.16.1
```

Comment:
🤔 does ops belong here? I'd imagine that docs are generated from the current repo.

Does it hurt? e.g. if someone is working on a branch, could this pull in different version of code?

Reply:
> 🤔 does ops belong here? I'd imagine that docs are generated from the current repo.

This file is auto-generated. ops ends up here because there's a circle: `ops-scenario` depends on `ops` (that could possibly be removed in time, but it would be a bunch of refactoring work) and `ops[testing]` depends on `ops-scenario`. The docs need to have `ops-scenario` installed to be able to pull in all the docstrings, and installing `ops-scenario` installs `ops`.

> Does it hurt? e.g. if someone is working on a branch, could this pull in different version of code?

Good question! It seems to work ok - I edited one of the ops modules locally and did `tox -e docs` and it picked up my local changes. I'm not totally sure about when readthedocs does the build - the config has:

```yaml
python:
  install:
    - method: pip
      path: .
      extra_requirements:
        - docs
```

So it's ignoring docs/requirements.txt I believe. I'm not sure what would happen if a branch of `ops` had a version number incompatible with the requirements in `ops-scenario` - probably an error? But that does seem like to be something we can manage ourselves, especially once we've merged the repos.